src/frontend/src/layouts/Sidebar/Sidebar.tsx

```diff
@@ -3,7 +3,13 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { faExchangeAlt, faFolder, faHome, faQuestionCircle, faUsers, faChartGantt } from '@fortawesome/free-solid-svg-icons';
+// Import MUI icons
+import HomeIcon from '@mui/icons-material/Home';
+import GanttIcon from '@mui/icons-material/BarChart';
```

Comment:
Can we use a different icon here? Perhaps `@mui/icons-material/AlignHorizontalLeft`?